libvcx/src/utils/devsetup.rs

```diff
@@ -1,17 +1,11 @@
 use std::fs;
 use std::sync::Once;
 
-use indy::future::Future;
-use indy::WalletHandle;
-use rand::Rng;
-use serde_json::Value;
-
-use crate::{api, init, libindy, settings, utils};
+use crate::{libindy, settings, utils};
```

Comment:
I wonder if `testsetup.rs` or something like that wouldn't be better name for the file

Reply:
sounds good to me, changed

Reply:
But this is making causing change in almost like every file :-D I'll leave this for some small separate PR rather.

Reply:
OK, it's not worth it in this PR :)